storage/mergerfs/mergerfs.yaml

```diff
@@ -0,0 +1,15 @@
+name: mergerfs
+depends:
+  - configuration: true
+container:
+  entrypoint: /usr/local/bin/mergerfs
+  args:
+  - -o config=/usr/local/etc/mergerfs/config.ini
```

Comment:
this should use an extensionserviceconfig so that the service waits on it, check the tailscale example

Reply:
I'm not at all sure, how this differs in that aspect of the tailscale example?

Reply:
@frezbo Did you not see it? https://github.com/skyssolutions/talos-extensions/blob/2d78d4eec93c7695b67e21a7ac3187a2be3e509d/storage/mergerfs/mergerfs.yaml#L3

Reply:
oh I missed that, sorry